tests/cnf/core/network/metallb/tests/bfd-test.go

```diff
@@ -502,7 +502,7 @@ func setLocalGWMode(status bool) {
 func verifyMetalLbBFDAndBGPSessionsAreUPOnFrrPod(frrPod *pod.Builder, peerAddrList []string) {
 	for _, peerAddress := range netcmd.RemovePrefixFromIPList(peerAddrList) {
 		Eventually(frr.BGPNeighborshipHasState,
-			time.Minute*3, tsparams.DefaultRetryInterval).
+			time.Minute*4, tsparams.DefaultRetryInterval).
```

Comment:
I remember you mentioning the change but not the reason? Any reason that it is taking longer?